packages/gatsby-plugin-gatsby-cloud/src/components/buttons/InfoIndicatorButton.js

```diff
@@ -61,6 +61,7 @@ const InfoIndicatorButton = ({
         ...btnProps,
         tooltip: {
           ...btnProps.tooltip,
+          overrideShow: visible,
```

Comment:
I tried using only show/override show instead of both, but there were some edge case scenarios that added weird behavior. Needs rework for sure- we dont need both of them